src/components/Input/Search.js

```diff
@@ -14,7 +15,11 @@ const Input = styled.input.attrs({
   width: 100%;
   padding: ${props =>
     props.slim
-      ? "9px 12px 9px calc(12px + 12px + 10px)"
+      ? customIf(
```

Comment:
const checkInvert = invert => invert ? "8px 12px 7px calc(12px + 12px + 10px)" : "9px 12px 8px calc(12px + 12px + 10px)";

padding: ${props =>
    props.slim
      ? checkInvert(props.invert) 
      : "14px 16px 14px calc(16px + 16px + 8px)"
    };

We can do it like this way too. So you do not require any new file and customIf.

Reply:
Firstly, I used this customIf to by-pass "nested ternary expressions not allowed" error from eslint. It seems like a better approach then disabling the eslint.
Secondly, there might come more situations in this project where we might have to use similar approach. So instead of having to re-write this expression again in every file i made a utility function.

Reply:
No, we can't create a separate for handling nested if-else. Use **styled-component** advance feature instead of it.

Reply:
Not entirely convinced because it isn't supporting re-usable code approach. still updating.